test/web-platform-tests/to-run.yaml

```diff
@@ -574,6 +574,17 @@ DIR: html/semantics/forms/resetting-a-form
 
 ---
 
+DIR: html/semantics/forms/textfieldselection
+
+select-event.html: [fail, not yet implemented]
+selection-after-content-change.html: [fail, not yet implemented]
+selection-not-application-textarea.html: [fail, not yet implemented]
+selection-start-end-extra.html: [fail, not yet implemented]
+textfieldselection-setRangeText.html: [fail, not yet implemented]
+textfieldselection-setSelectionRange.html: [fail, not yet implemented]
+
```

Comment:
Which tests in this directory fail before your change but pass afterward?

Reply:
The following now pass:
```
selection-not-application.html
selection-start-end.html
selection-value-interactions.html
selection.html
setSelectionRange.html
textarea-selection-while-parsing.xhtml
```

Reply:
`selection-after-content-change.html` also passes after the change to use `_getAPIValue` to match spec.